app/controllers/application_controller.rb

```diff
@@ -1,19 +1,15 @@
 class ApplicationController < ActionController::Base
-  rescue_from DeviseShibbolethAuthenticatable::ShibbolethException do |exception|
+  rescue_from DeviseAafRcAuthenticatable::AafRcException do |exception|
     render :text => exception, :status => 500
   end
-  prepend_before_filter :retrieve_aaf_credentials
 
   before_filter :shib_sign_up_redirect, :except => :root
```

Comment:
Are references to shib still relevant? if not then please change since it will be confusing